packages/flutter/test/material/tabs_test.dart

```diff
@@ -1277,8 +1277,8 @@ void main() {
   testWidgets('Overflowing RTL tab bar', (WidgetTester tester) async {
     final List<Widget> tabs = new List<Widget>.filled(100,
       // For convenience padded width of each tab will equal 100:
-      // 76 + kTabLabelPadding.horizontal(24)
-      new SizedBox(key: new UniqueKey(), width: 76.0, height: 40.0),
+      // 68 + kTabLabelPadding.horizontal(32)
```

Comment:
This test depends extensively on the "width of each tab will equal 100", per line 1279. It would be best to restore that invariant and make sure everything still works without changing the other dimensions around.

Reply:
Sorry about that. Apparently I can't add :-)